schema/workflow.yaml

```diff
@@ -767,7 +767,7 @@ $defs:
     unevaluatedProperties: false
     properties:
       set:
-        type: object
+        type: [object, string]
```

Comment:
For the sake of consistency, could you please use a `oneOf` instead of a type array?